bin/create_training_data.py

```diff
@@ -3,45 +3,84 @@
 import argparse
 
 from src.download_naips import NAIPDownloader
-from src.create_training_data import (NUMBER_OF_NAIPS, RANDOMIZE_NAIPS, NAIP_STATE, NAIP_RESOLUTION,
-                                      NAIP_YEAR, NAIP_SPECTRUM, NAIP_GRID, HARDCODED_NAIP_LIST,
-                                      random_training_data, equalize_data, split_train_test, 
+from src.create_training_data import (random_training_data, equalize_data, split_train_test, 
                                       format_as_onehot_arrays, dump_data_to_disk)
 
-
 def create_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("--tile-size",
                         default=64,
                         type=int,
                         help="tile the NAIP and training data into NxN tiles with this dimension")
+    parser.add_argument("--tile-overlap",
+                        default=1,
+                        type=int,
+                        help="divide the tile-size by this number for how many pixels to move over when tiling data"
+                        "this is set to 1 by default, so tiles don't overlap"
+                        "setting it to 2 would make tiles overlap by half, and setting it to 3 would make the tiles overlap by 2/3rds")
+    parser.add_argument("--pixels-to-fatten-roads",
+                        default=3,
+                        type=int,
+                        help="the number of pixels to fatten a road centerline (e.g. the default 3 makes roads 7px wide)")
+    parser.add_argument("--percent-for-training-data",
+                        default=.90,
+                        type=float,
+                        help="how much data to allocate for training, the remainder is left for test")
     parser.add_argument("--band-list",
                         default=[0, 0, 0, 1],
                         nargs=4,
                         type=int,
-                        help="specify which bands to activate (R  G  B  IR). default is "
+                        help="specify which bands to activate (R  G  B  IR)"
                         "--bands 0 0 0 1 (which activates only the IR band)")
+    parser.add_argument("--naip-path",
+                        default=['md', '2013', '1m', 'rgbir', '38077'],
+                        nargs=5,
+                        type=str,
+                        help="values to create the S3 bucket path for some NAIPs"
+                        "--naip-path md 2013 1m rgbir 38077 (defaults to some Maryland data)")
+    parser.add_argument("--randomize-naips",
+                        default=False,
+                        type=bool,
```

Comment:
Instead of `default=False, type=bool` if you do `action='store_true'` then you don't have to do `--naip-path=True` on the command line. You can just do `--naip-path` (or name it something like `--enable-naip-path`) for true and omit it for false (which is how most cmd line flags work).